pkg/api/server/v1alpha2/results_test.go

```diff
@@ -147,6 +149,112 @@ func TestGetResult(t *testing.T) {
 	}
 }
 
+func TestUpdateResult(t *testing.T) {
+	srv, err := New(test.NewDB(t))
+	if err != nil {
+		t.Fatalf("failed to create server: %v", err)
+	}
+	ctx := context.Background()
+
+	tt := []struct {
+		name      string
+		parent    string
+		in        *pb.Result
+		fieldmask *field_mask.FieldMask
+		update    *pb.Result
+		expect    *pb.Result
+		errcode   codes.Code
+	}{
+		{
+			name:   "Test no Mask",
+			in:     &pb.Result{Name: "foo/results/bar-001"},
+			update: &pb.Result{Annotations: map[string]string{"foo": "bar"}},
+			expect: &pb.Result{Annotations: map[string]string{"foo": "bar"}},
```

Comment:
It wasn't intuitive on first pass that that this was a partial response. We should add a comment above explaining the behavior.

Reply:
No masks, so done.